substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2186,6 +2186,41 @@ impl pallet_broker::Config for Runtime {
 	type PriceAdapter = pallet_broker::CenterTargetPrice<Balance>;
 }
 
+parameter_types! {
+	pub const PotId: PalletId = PalletId(*b"py/potid");
+	pub const Period:BlockNumber = 5*MINUTES;
```

Comment:
```suggestion
	pub const Period:BlockNumber = 5 * MINUTES;
```

`cargo fmt` don't work well with code inside macro